examples/hello_world/policies/Watchlog.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
move this one to the module and attached it always as we will need logging capabilities. You can use [iam_policy_document](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document)

Reply:
Done, commit: 79e714e4a1477d97d31c1a916d9f2c09e3bd527d